app/services/submission/SubmissionService.scala

```diff
@@ -240,7 +234,7 @@ class SubmissionService @Inject() (
       identificationNumber <- (__ \ TransportMeansIdentificationNumberPage(index).toString).read[String]
       nationality          <- (__ \ TransportMeansNationalityPage(index).toString).read[Nationality]
     } yield DepartureTransportMeansType05(
-      sequenceNumber = index.sequenceNumber,
+      sequenceNumber = BigInt(index.sequenceNumber),
```

Comment:
Is there any reason for the `sequenceNumber` val inside `Index` to remain a `String`? Could we just have `val sequenceNumber: BigInt = display`? Then there would be no need to wrap everything in a `BigInt` here.